src/leveled_runner.erl

```diff
@@ -671,49 +644,43 @@ accumulate_objects(FoldObjectsFun, InkerClone, Tag, DeferredFetch) ->
             % a fold_objects), then a metadata object needs to be built to be
             % returned - but a quick check that Key is present in the Journal
             % is made first
-            case leveled_codec:is_active(LK, V, Now) of
-                true ->
-                    {SQN, _St, _MH, MD} =
-                        leveled_codec:striphead_to_v1details(V),
-                    {B, K} =
-                        case leveled_codec:from_ledgerkey(LK) of
-                            {B0, K0} ->
-                                {B0, K0};
-                            {B0, K0, _T0} ->
-                                {B0, K0}
-                        end,
-                    JK = {leveled_codec:to_ledgerkey(B, K, Tag), SQN},
-                    case DeferredFetch of
-                        {true, JournalCheck} ->
-                            ProxyObj = 
-                                leveled_codec:return_proxy(Tag, MD,
-                                                            InkerClone, JK),
-                            case JournalCheck of
-                                true ->
-                                    InJournal =
-                                        leveled_inker:ink_keycheck(InkerClone,
-                                                                    LK,
-                                                                    SQN),
-                                    case InJournal of
-                                        probably ->
-                                            FoldObjectsFun(B, K, ProxyObj, Acc);
-                                        missing ->
-                                            Acc
-                                    end;
-                                false ->
```

Comment:
This line suggests that JournalCheck always is a boolean value, otherwise this would have crashed... but in line 628 it is suggested that JournalCheck could have a different value than false, otherwise why not doing an "and" and check true and false there?

Reply:
Clarified this, and added types to help dialyzer going forward